### PR TITLE
UINV-14 Settings: Manage pre-set adjustments

### PR DESCRIPTION
**Pull request by aliaksei-chumakou**
## Purpose
ability to create an invoice line that details the value and what is being paid for by the invoice
https://issues.folio.org/browse/UINV-14

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/settings/adjustments/SettingsAdjustments.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    stripes: stripesShape.isRequired,
+    resources: PropTypes.object,
+  };
+
+  constructor(props) {
+    super(props);
+
+    const { stripes } = props;
+
+    this.connectedSettingsAdjustmentsList = stripes.connect(SettingsAdjustmentsList);
```
Comment (NikitaSedyx) [inline src/settings/adjustments/SettingsAdjustments.js]: it can be imported with connect

File: src/settings/adjustments/SettingsAdjustments.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+          />
+          <Route
+            path={`${path}/:id/view`}
+            render={(props) => (
+              <this.connectedSettingsAdjustmentsView
+                {...props}
+                close={this.closePane}
+                rootPath={path}
+                showSuccessDeleteMessage={this.showSuccessDeleteMessage}
+                stripes={stripes}
```
Comment (NikitaSedyx) [inline src/settings/adjustments/SettingsAdjustments.js]: stripes is not required with HoC usage

File: src/settings/adjustments/SettingsAdjustmentsEditor/SettingsAdjustmentsEditor.js [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+
+    return (
+      <PaneMenu>
+        <FormattedMessage id="ui-invoice.save">
+          {ariaLabel => (
+            <Button
+              id="save-adjustment-button"
+              type="submit"
+              disabled={pristine || submitting}
+              style={{ marginBottom: '0', marginRight: '10px' }}
```
Comment (NikitaSedyx) [inline src/settings/adjustments/SettingsAdjustmentsEditor/SettingsAdjustmentsEditor.js]: styles file?

Comment (aliaksei-chumakou) [reply src/settings/adjustments/SettingsAdjustmentsEditor/SettingsAdjustmentsEditor.js]: I'll create a tech-debt for it

Comment (aliaksei-chumakou) [reply src/settings/adjustments/SettingsAdjustmentsEditor/SettingsAdjustmentsEditor.js]: https://issues.folio.org/browse/UINV-46

**Comment by aliaksei-chumakou** [comment]
tests will be added later. within UINV-8

**Pull request merged**